cypress/e2e/EngageSphere_A11Y.cy.js

```diff
@@ -0,0 +1,27 @@
+//Tabela de clientes
+
+import 'cypress-axe';
```

Comment:
Recomendo deixar uma linha em branco entre a importação de _libs_ externas e outras partes do código.

```suggestion
import 'cypress-axe';

```